js/core/dom_component.js

```diff
@@ -360,6 +396,19 @@ var DOMComponent = Component.inherit({
     */
     element: function() {
         return this._$element;
+    },
+
+    /**
+    * @name domcomponentmethods_dispose
+    * @publicName dispose()
```

Comment:
It seems that you should update TypeScript definitions also

Reply:
I have added it https://github.com/DevExpress/DevExtreme/pull/1261/files#diff-febb761b8ab6b2590bab0e6586f5a5e2R434